### DISCUSSION TITLE
Can SMP characteristic use Write With Response?

### DISCUSSION THREAD

**Discussion by JPHutchins** [zephyrproject-rtos/zephyr#73623] [Ideas]
Are there any reasons not to add a Kconfig that enables write with response on the SMP characteristic?

https://github.com/zephyrproject-rtos/zephyr/blob/fe03e0c4bb3f8a658efe69b04c430d1a15474c6c/subsys/mgmt/mcumgr/transport/src/smp_bt.c#L356-L368

Working through some compatibility issues with MacOS but I suspect it's a symptom of a bigger problem.

https://github.com/hbldh/bleak/discussions/1589

https://github.com/intercreate/smpclient/issues/30

FYI @nordicjm @fabiobaltieri

**Comment by fabiobaltieri** [comment]
cc @de-nordic

**Comment by nordicjm** [comment]
Yes there is because it uses notifications, not indications.
